src/sgame/sg_main.cpp

```diff
@@ -190,6 +190,8 @@ vmCvar_t           g_instantBuilding;
 
 vmCvar_t           g_emptyTeamsSkipMapTime;
 
+Cvar::Cvar<bool>   g_neverEnd("g_neverEnd", "cheat to never end a game, helpful to load a map without spawn for testing purpose", Cvar::NONE, false);
+
```

Comment:
FYI: You could also place the declaration just above the function where it is used. No need to change it though